src/main/java/frc/robot/RobotContainer.java

```diff
@@ -41,22 +42,6 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    // Schedule `ExampleCommand` when `exampleCondition` changes to `true`
-    new Trigger(m_exampleSubsystem::exampleCondition)
-        .onTrue(new ExampleCommand(m_exampleSubsystem));
-
-    // Schedule `exampleMethodCommand` when the Xbox controller's B button is pressed,
-    // cancelling on release.
-    m_driverController.b().whileTrue(m_exampleSubsystem.exampleMethodCommand());
-  }
-
-  /**
-   * Use this to pass the autonomous command to the main {@link Robot} class.
-   *
-   * @return the command to run in autonomous
-   */
-  public Command getAutonomousCommand() {
-    // An example command will be run in autonomous
-    return Autos.exampleAuto(m_exampleSubsystem);
+    m_driverController.a().whileTrue(new InstantCommand(intake::deploy, intake));
```

Comment:
For readability, I'm a big fan of what we did last year:
```java
driverController
    .TriggerLeft()
    .whileActiveContinuous(
        new InstantCommand(intake::intakeCargo, intake).withName("Intaking"));
```
It'd be nice if we stick with that again. No reason to keep it on one line, and being able to at a glance see which controller, which control, and the action separately is really nice.

Reply:
Yeah, I'd agree that format is much better for read/write usage